systemlink-notebook-datasource/src/DataSource.ts

```diff
@@ -145,6 +149,8 @@ export class DataSource extends DataSourceApi<NotebookQuery, NotebookDataSourceO
 
         frames.push(frame);
       }
+    } else if (result.type === 'array') {
+      frames.push(new MutableDataFrame({ refId: query.refId, fields: [{ name: result.id, values: result.data }] }));
```

Comment:
Is this new array type only "defined" here? Or is there documentation/examples somewhere else as well?

Reply:
It's only defined here. I'm planning on making an example notebook to give to MCS when they start implementing dashboards.